exercises/random/challenge_053.py

```diff
@@ -0,0 +1,14 @@
+### RANDOM CHALLENGE ###
+
+"""
+053
+Display a random fruit from a list of five fruits
+"""
+import random
+
+
+def get_random_choice(input_array):
```

Comment:
This abstraction isn't adding a lot of clarity to the code

Reply:
I have added the method to make testing easier.

Reply:
That's a sensible reason but also check the `patch` documentation: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch